GDJS/package.json

```diff
@@ -11,7 +11,6 @@
     "@types/node": "^14.11.1",
     "@types/sinon": "^10.0.13",
     "@types/three": "0.152.0",
-    "better-docs": "^2.3.2",
```

Comment:
I checked, jsdoc is not used since this commit: https://github.com/4ian/GDevelop/commit/2c4fa1d1c20c40d4fca8316c2076d39a6e96cdfa
So safe to remove